build/run-backup.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e
+
+request_backup() {
+	curl -s http://${SRC_NODE}:6033/backup/${BACKUP_NAME} -o /backup/${BACKUP_NAME}.stream
+}
+
+request_logs() {
+	curl -s http://${SRC_NODE}:6033/logs/${BACKUP_NAME}
+}
+
+run_s3() {
+	cat /backup/${BACKUP_NAME}.stream \
+		| xbcloud put ${BACKUP_NAME} --storage=s3 --s3-bucket="${S3_BUCKET}"
```

Comment:
We need to have name like `demand-backup-2022-05-25-11:26:11-full` but not name of the backup. This name is not uniq.

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/77/commits/94702b630e47fc6ded4c8b273d197dfce0d1d2d7